mutatex/core.py

```diff
@@ -1058,15 +1081,35 @@ def partial_clean(self):
 
     def reset_working_directory(self):
         """
-        no directory reset needs to performed for for Repair runs, so this
-        function will always return False
+        Performs complete reset of the working directory. This function
+        removes all the input and output files from the previous run. 
+        This is done in case of unrecognized molecules.
+        
         Returns
         ----------
-        False
-            return values that signals the prepare function that the directory
-            hasn't ben reset
+        True or False: bool 
```

Comment:
still not the expected format :) it should be at least

```
type(s)
    my description
```
  
and so:
    
```
bool
    True if ..., otherwise False
```